queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -109,7 +109,7 @@ def get_message_context_properties(queue_msg: nats.aio.client.Msg):
                      'restoration', 'specialResolution', 'correction', 'amalgamationApplication', 'continuationIn') \
                 and (option := email.get('option', None)) \
                 and (filing_id := email.get('filingId', None)):
-            # option contains current status of filing - PAID or COMPLETED
+            # option contains current status of filing - PAID or COMPLETED or CHANGE_REQUESTED, etc
```

Comment:
I added this because it's unlike the other cases above where `option` is a state filter (eg, line 102).

We'll be dealing with CHANGE_REQUESTED emails in a future ticket (22363).